.github/workflows/sync.yml

```diff
@@ -0,0 +1,66 @@
+name: sync-schedule
+on:
```

Comment:
In this GHA only the compare will be ran on a schedule right? And then the owner of the Task-Registry should run the script/sync -sync?

Reply:
Yes.

Reply:
ah, this answers the question I asked below